.travis.yml

```diff
@@ -7,4 +7,4 @@ services:
 
 script:  
   - npm ci
-  - docker run hello-world
\ No newline at end of file
+  - npm run playwright-testing-lint
```

Comment:
This linting is not specific to playwright. This should just be `npm run lint`

Reply:
Noted @vidhi-mody